### PR TITLE
Added database module. Added api route to handle url queries.

### PR DESCRIPTION
**Pull request by midfords**
Created a database module. Use this for all database access.
Created an api route to answer url queries for the mobile app.

### CODE REVIEW

File: webapp/npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,36 @@
+0 info it worked if it ends with ok
```
Comment (paymog) [inline webapp/npm-debug.log]: Should be gitignored

File: webapp/routes/api.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+// This route is to serve json
+//
+
+var express = require('express');
+var database = require('../database')
+var router = express.Router();
+
+router.get('/', function(req, res) {
+
+    var db_res = database.createUser(req.query.usr)
```
Comment (paymog) [inline webapp/routes/api.js:11]: XSS vulnerability. Should not create users on a get but instead on a post.

File: webapp/routes/index.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
 var express = require('express');
 var router = express.Router();
 
-/* GET home page. */
```
Comment (paymog) [inline webapp/routes/index.js:4]: Why?

File: webapp/server.js [hunk 4]
```diff
@@ -1,6 +1,10 @@
 var http = require('http');
```
Comment (paymog) [inline webapp/server.js:1]: Why was this file touched?

**Comment by midfords** [comment]
Fixed XSS, changed to router.post
Added npm-debug.log to .gitignore.

**Pull request closed without merging**